iwinfo_lua.c

```diff
@@ -279,6 +279,9 @@ static void set_rateinfo(lua_State *L, struct iwinfo_rate_entry *r, bool rx)
 	lua_pushboolean(L, r->is_he);
 	lua_setfield(L, -2, rx ? "rx_he" : "tx_he");
 
+	lua_pushboolean(L, r->is_eht);
+	lua_setfield(L, -2, rx ? "rx_eht" : "tx_eht");
+
 	lua_pushnumber(L, r->mhz);
```

Comment:
Assuming the updated structure layout, this should become 
`lua_pushnumber(L, r->mhz_hi * 256 + r->mhz);`